docs/components/ComponentLayout.css

```diff
@@ -75,6 +75,10 @@ ul.Component__list li {
   overflow: hidden;
 }
 
+.Component .Offscreen {
+  top: 0;
+}
```

Comment:
I noticed that Offscreen was creating double scrollbars on this page as it was positioned outside of the scrollable area creating some weird jank.